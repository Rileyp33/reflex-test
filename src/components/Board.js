import React from 'react'
import {
  View,
  Dimensions,
  Pressable
} from 'react-native'
import { ScaledSheet } from 'react-native-size-matters'

import Card from './Card'
import { times } from '../helpers'

const Board = ({
  activeSquares,
  activeColor,
  gameActive,
  handlePress,
  interval
}) => {
  const renderGameButton = (i) => {
    const containerWidth = Dimensions.get('window').width - 32
    const buttonWidth = containerWidth / 4
    const isActive = activeSquares.includes(i)
    return (
      <Pressable
        key={i}
        onPressIn={() => handlePress(i)}
        disabled={!gameActive}
        style={{
          width: buttonWidth,
          height: buttonWidth,
        }}
      >
        {({ pressed }) => (
          <Card
            flipped={isActive && activeColor}
            isActive={isActive}
            i={i}
            interval={interval}
            pressed={pressed}
          />
        )}
      </Pressable>
    )
  }

  return (
    <View style={styles.boardContainer}>
      {times(16, renderGameButton)}
    </View>
  )
}

const styles = ScaledSheet.create({
  boardContainer: {
    flexWrap: 'wrap',
    flexDirection: 'row'
  },
  image: {
    width: '100%',
    height: '100%',
    resizeMode: 'cover'
  }
})

export default Board

