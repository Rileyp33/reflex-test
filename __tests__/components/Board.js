import React from 'react'
import renderer from 'react-test-renderer'
import Board from '../../src/components/Board'

test('Game board renders correctly in difficulty level 1 on an odd numbered turn with the last square purple', () => {
  const tree = renderer.create(
    <Board
      activeSquares={[15]}
      activeColor={true}
      gameActive={true}
      handlePress={() => { }}
      turns={1}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Game board renders correctly in difficulty level 2 on an even numbered turn with the first two squares blue', () => {
  const tree = renderer.create(
    <Board
      activeSquares={[0, 1]}
      activeColor={true}
      gameActive={true}
      handlePress={() => { }}
      turns={0}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})

test('Game board renders correctly in difficulty level 3 on an odd numbered turn with the last three squares purple', () => {
  const tree = renderer.create(
    <Board
      activeSquares={[13, 14, 15]}
      activeColor={true}
      gameActive={true}
      handlePress={() => { }}
      turns={1}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
