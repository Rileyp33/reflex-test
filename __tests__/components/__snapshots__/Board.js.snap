// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game board renders correctly in difficulty level 1 on an odd numbered turn with the last square purple 1`] = `
<View
  style={
    Object {
      "borderColor": "#6e6e6e",
      "borderWidth": 1,
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#009FB7",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
</View>
`;

exports[`Game board renders correctly in difficulty level 2 on an even numbered turn with the first two squares blue 1`] = `
<View
  style={
    Object {
      "borderColor": "#6e6e6e",
      "borderWidth": 1,
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#9B5DE5",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#9B5DE5",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
</View>
`;

exports[`Game board renders correctly in difficulty level 3 on an odd numbered turn with the last three squares purple 1`] = `
<View
  style={
    Object {
      "borderColor": "#6e6e6e",
      "borderWidth": 1,
      "flexDirection": "row",
      "flexWrap": "wrap",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#009FB7",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#009FB7",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#009FB7",
        "borderColor": "#E6E6EA",
        "borderWidth": 0.5,
        "height": 179.5,
        "opacity": 1,
        "width": 179.5,
      }
    }
  />
</View>
`;
